test/libsolidity/syntaxTests/array/bytes_push_assign_multi.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    bytes x;
+    bytes z;
+    function f() public {
+        (x.push(), x.push()) = (0,0);
+        (((x.push())), (x.push())) = (0,0);
+        ((x.push(), x.push()), x.push()) = ((0,0), 0);
+        (x.push(), x[0]) = (0, 0);
+        bytes storage y = x;
+        (x.push(), y.push()) = (0,0);
+        // The following is a false positive.
+        (x.push(), z.push()) = (0,0);
```

Comment:
Not sure if we should be bothered by such false positives or not.
For comparison: the other warning regarding copies to storage has similar false positives - it'll warn about ``(x,y) = (y,x)``, but also about ``(x,y) = (z, w)``.